backend/simple_final_scoring.py

```
@@ -0,0 +1,238 @@
+        prd_scorer = PRDScoringService()
+        prd_result = prd_scorer.score_guide(guide_data, destination)
+        scores['prd_compliance'] = prd_result.get('overall_score', 0)
+    except Exception as e:
+        print(f"⚠️ PRD Scoring failed: {e}")
+        scores['prd_compliance'] = 0
+    
+    try:
+        harsh_scorer = HarshPRDScorer()
+        harsh_result = harsh_scorer.score_guide(guide_data, destination)
+        scores['luxury_standards'] = harsh_result.get('overall_score', 0)
+    except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)